apps/atomy-q/API/app/Http/Controllers/Api/V1/AwardController.php

```
@@ -16,10 +22,19 @@
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Str;
+use Nexus\Common\Contracts\ClockInterface;
+use Nexus\IntelligenceOperations\DTOs\AiStatusSchema;
 
 final class AwardController extends Controller
 {
     use ExtractsAuthContext;
+    use InteractsWithAiAvailability;
+
+    public function __construct(
+        private readonly ComparisonAwardAiClientInterface $comparisonAwardAiClient,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $comparisonAwardAiClient. Keep variable name length under 20. (undefined)

(LongVariable)